packages/flutter_tools/templates/plugin/lib/projectName.dart.tmpl

```diff
@@ -6,6 +6,8 @@ class {{pluginDartClass}} {
   static const MethodChannel _channel =
       const MethodChannel('{{projectName}}');
 
-  static Future<String> get platformVersion =>
-      _channel.invokeMethod('getPlatformVersion');
+  static Future<String> get platformVersion async {
+    final String version = await _channel.invokeMethod('getPlatformVersion');
+    return version;
```

Comment:
Why was this change necessary? Is there a bug? Is it related to https://github.com/dart-lang/sdk/issues/31743 ?

Reply:
`invokeMethod` returns `Future<dynamic>` not `Future<String>`.